Gruntfile.js

```diff
@@ -106,7 +107,20 @@ module.exports = function (grunt) {
         }
       }
     },
-
+      critical: {
+          test: {
+              options: {
+                  base: './',
+                  css: ['web/css/landing.opentok.css'],
+                    target: {
+                      uncritical: 'web/css/landing-uncritical.opentok.css',
+                    },
+                      minify: true
+              },
+              src: 'views/index.ejs',
+              dest: 'web/css/landing-critical.opentok.css'
+          }
+      },
```

Comment:
These lines are indented too much (they should use 2 spaces for indentation).

Reply:
Formatted the task